awx/playbooks/project_update.yml

```diff
@@ -240,12 +240,34 @@
       with_fileglob:
         - "{{ project_path | quote }}/collections/requirements.yaml"
         - "{{ project_path | quote }}/collections/requirements.yml"
-        - "{{ project_path | quote }}/requirements.yaml"
-        - "{{ project_path | quote }}/requirements.yml"
       changed_when: "'Installing ' in galaxy_collection_result.stdout"
       environment: "{{ additional_collections_env | combine(galaxy_task_env) }}"
       when:
         - "ansible_version.full is version_compare('2.9', '>=')"
         - collections_enabled | bool
       tags:
         - install_collections
+
+    - name: Fetch galaxy roles and collections from requirements.(yml/yaml)
+      ansible.builtin.command: >
+        ansible-galaxy install -r {{ item }}
+        {{ ' -' + 'v' * ansible_verbosity if ansible_verbosity else '' }}
+      args:
+        chdir: "{{ project_path | quote }}"
+      register: galaxy_combined_result
+      with_fileglob:
+        - "{{ project_path | quote }}/requirements.yaml"
+        - "{{ project_path | quote }}/requirements.yml"
+      changed_when: "'Nothing to do.' not in galaxy_combined_result.stdout"
+      environment: "{{ additional_collections_env | combine(galaxy_task_env) | combine(paths) }}"
+      vars:
+        paths:
+          ANSIBLE_COLLECTIONS_PATHS: "{{ projects_root }}/.__awx_cache/{{ local_path }}/stage/requirements_collections"
```

Comment:
pretty sure the plural here is going to be deprecated soon. Maybe we want to be forward thinking?

Reply:
@gamuniz that's pre-existing. Sure, rename it, but do it in the right place.

Here, it is not making any sense why ` | combine(paths)` is added. That is, why was the `paths` variable introduced at all? It appears to be the same as `additional_collections_env`. This is, again, duplicated with the use of `--roles-path` and `--collections-paths` CLI flags in specific commands (but I would not touch this right now). This also shouldn't come at higher precedence than the user's custom `galaxy_task_env` setting.

Reply:
In, at least my version of ansible, ansible-galaxy install does not respect the --collections-paths:
```
$ ansible-galaxy install --collections-path . awx.awx
usage: ansible-galaxy [-h] [--version] [-v] TYPE ...
ansible-galaxy: error: unrecognized arguments: --collections-path
```

But it does respect the ANSIBLE_COLLECTIONS_PATHS environment variable and combine takes a dict so I just make the `paths` dict variable to do the combine to make the generic install work.

Reply:
@gamuniz do you know where the deprecation message is? I see them both in the [latest docs](https://docs.ansible.com/ansible/latest/reference_appendices/config.html) but don't see a deprecation warning on either. I could be looking in the wrong spot though.